app/src/main/java/com/tma/sparking/services/http/RetrofitParkingFieldService.java

```diff
@@ -14,28 +11,15 @@
 import retrofit2.converter.gson.GsonConverterFactory;
 
 /**
- * Service for reading data from server
+ * An implement of ParkingFieldService using Retrofit
  */
-public class ParkingFieldService {
+public class RetrofitParkingFieldService implements ParkingFieldService {
     private static final String FIELD_PREFIX = "field";
```

Comment:
This is duplicate with FIELD_PREFIX in GsonParser class. Please remove the duplication